code/game/machinery/transportpod.dm

```diff
@@ -0,0 +1,121 @@
+/obj/machinery/transportpod
+	name = "Ballistic Transportation Pod"
+	desc = "A fast transit ballistic pod used to get from one place to the next. Batteries not included!"
+	icon = 'icons/obj/structures.dmi'
+	icon_state = "borg_pod_opened"
+
+	density = 1 //thicc
+	anchored = 1
+	use_power = 0
+
+	var/used = 0
+	var/in_transit = 0
+	var/mob/occupant = null
+
+	var/xc = list(137, 209, 163, 110, 95, 60, 129, 201) // List of x values on the map to go to.
+	var/yc = list(134, 99, 169, 120, 96, 122, 189, 219) // List of y values on the map to go to.
```

Comment:
So this is just a list of 8 specific locations you can get sent to?

Reply:
Yes.

Reply:
Roughly where are these locations?

Reply:
Uhr.. Cargo area, security, engineering, around primary tool storage (the one near technical storage, I think), Medical..

Reply:
Don't remember all the locations, just those general ones.